### DISCUSSION TITLE
package name

### DISCUSSION THREAD

**Discussion by kyle-messier** [NIEHS/chopin#17] [General]
@sigmafelix I'm curious where you came up with the name SCOMPS? Any other potential ideas?

**Comment by sigmafelix** [comment]
For naming the package, [s]calable [comp]utation for geo[s]patial health studies was my initial thought, but I will be happy to discuss a proper name. Other quick thoughts are--

- I used a name for the similar package for spatial computation `scoR` (spatial computation in R or scalable spatial computation in R)
- `scog` (scalable computation for geohealth research) (dictionary: _dialect Scot_ to shelter.)
- `sachet`(scalable/spatial computation for health-environment research toolkit) (dictionary: a small perfumed bag used to scent clothes.)
